src/ASM/LR/ASMu3D.C

```diff
@@ -1751,12 +1751,12 @@ bool ASMu3D::evalSolution (Matrix& sField, const IntegrandBase& integrand,
     fe.v   = gpar[1][i];
     fe.w   = gpar[2][i];
 
-    double u[2*p1];
-    double v[2*p2];
-    double w[2*p3];
-    basis1.computeBasisValues(gpar[0][i], u, 1);
-    basis2.computeBasisValues(gpar[1][i], v, 1);
-    basis3.computeBasisValues(gpar[2][i], w, 1);
+    std::vector<double> u(2*p1);
+    std::vector<double> v(2*p2);
+    std::vector<double> w(2*p3);
```

Comment:
should then have the definition on one line `std::vector<double> u(2*p1), v(2*p2), w(2*p3)` like in L2451 below

Reply:
done